lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -103,6 +103,11 @@ llvm::cl::list<std::string>
                        llvm::cl::desc("Verify the module/analyses after we run "
                                       "a pass from this list"));
 
+llvm::cl::list<std::string>
+    SILForceVerifyAfterPass("sil-force-verify-after-pass",
```

Comment:
This name is really misleading. Can you rename to: sil-verify-all-force-precompute-analysis. Make sure that the description is clear that the option forces the pre-computation of analyses.

Reply:
Changed it to -sil-force-precompute-verify-analysis